os/dutils.sh

```diff
@@ -243,12 +244,21 @@ function MENU()
 function TOOLCHAIN()
 {
 	ARGS=arm-none-eabi-$2
+
+	OUTPUTFILE+="$2.txt"
+	TIMESTAMP=$(date "+%Y.%m.%d-%H.%M.%S")
+	OUTPUTFILE=$OUTPUTFILE.$TIMESTAMP
```

Comment:
How about formatting like this?
```
COMMAND_$TIMESTAMP.txt
```
The COMMAND could be `nm`, `addr2line` or manual input. (similar as Abhishek suggested)